doc/guide/feature-selinux.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+	"http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+<chapter id="feature-selinux">
+  <title>SELinux Policy</title>
+
+  <para>If present on the system Cockpit can enable and disable SELinux. It can also
```

Comment:
Okay, will adjust this. But feel free to do follow ups too, to make this page more complete. Updating the documentation should have been a blocker for getting the initial SELinux pull request in.